realm/realm-annotations-processor/src/test/resources/io/realm/some_test_AllTypesRealmProxy.java

```diff
@@ -868,6 +868,10 @@ public static String getSimpleClassName() {
         return "AllTypes";
     }
 
+    public static final class ClassNameHelper {
```

Comment:
why using an inner class can't you store it/access using a static field only instead?